packages/protobuf-test/src/gen/js/google/protobuf/struct_pb.js

```diff
@@ -113,7 +113,12 @@ Value.prototype.toJson = function toJson(options) {
     case "nullValue":
       return null;
     case "boolValue":
+      return this.kind.value;
     case "numberValue":
+      if (!isFinite(this.kind.value)) {
```

Comment:
Best to use the explicit Number.isFinite here.

Previously, the switch statement matched three cases to limit the number of returns required. Can we retain this? Fine to have a fall-through here, we do it in other places as well.